### DISCUSSION TITLE
MPI warning on Hamilton8

### DISCUSSION THREAD

**Discussion by shadowbourne** [wenceorg/comp4187#93] [Q&A] [answered]
Atm code seems to run fine..?, however I get this warning every time i run a .py script:

[0] MPI startup(): FI_PSM3_UUID was not generated, please set it to avoid possible resources ownership conflicts between MPI processes

Is this expected / alarming? (I've had alot of issues getting petsc to install using your instructions, it finally worked but now a warning :( )

**Comment by annereinarz** [comment]
Just to double-check, are you using the modules in the version numbers suggested in the installation guide?

**Comment by shadowbourne** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/18665030/165174215-0a01858d-7dee-429d-9dc2-6e147a4c92f7.png)
What version numbers?

**Reply by annereinarz** [reply]
Sorry, its the default versions of everything as you say. Can you check you've got:

Currently Loaded Modulefiles:
  1) gcc/11.2          2) intelmpi/2021.4   3) openblas/0.3.18   4) python/3.9.9

**Reply by shadowbourne** [reply]
I do indeed.

**Reply by annereinarz** [reply]
Then I'm at a bit of a loss as to where that message could be coming from. I've just reinstalled following the instructions exactly and don't see this message. Have you done any steps slightly differently? Did you get any errors while building petsc? Are you running this with slurm? If so can you show me the jobscript?

**Reply by shadowbourne** [reply]
I am also at a loss, but I reinstalled four times to get this to work so I'm going to leave it to be for now, as things seem to run fine in serial and parallel? Thus for now I will mark this as resolved.
